src/functions/formatSort.js

```diff
@@ -5,22 +5,29 @@ var find = require('./find');
 /**
  * Transform sort format from user friendly notation to lodash format
  * @param {string[]} sortBy array of predicate of the form "attribute:order"
+ * @param {string[]} [defaults] array of predicate of the form "attribute:order"
  * @return {array.<string[]>} array containing 2 elements : attributes, orders
  */
 module.exports = function formatSort(sortBy, defaults) {
+  defaults = (defaults || []).map(function(sort) {
+    return sort.split(':');
+  });
+
   return sortBy.reduce(
-    function preparePredicate(out, sortInstruction) {
-      var sortInstructions = sortInstruction.split(':');
-      if (defaults && sortInstructions.length === 1) {
-        var similarDefault = find(defaults, function(predicate) {
-          return predicate[0] === sortInstruction[0];
+    function preparePredicate(out, sort) {
+      var sortInstruction = sort.split(':');
+      if (sortInstruction.length === 1) {
```

Comment:
It would ease the reading with the condition inverted. We can early return at the beginning of the function, it would then reduce the nesting of the function.